### DISCUSSION TITLE
Support for HTML abbr tags

### DISCUSSION THREAD

**Discussion by cpbr-dev** [scalar/scalar#1378] [Ideas]
Would love to see support for [HTML abbreviation elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr#providing_an_expansion), specifically support for the title attribute.
I'm currently reworking API documentation for better accessibility and multiple acronyms are being used there which could benefit from this.

**Comment by marclave** [comment]
hi @cpbr-dev ! let's add this :) thanks for making the discussion

**Reply by hanspagel** [reply]
Just for reference: It’s already in. https://github.com/scalar/scalar/pull/1860 :)
